sentry_streams/tests/test_pipeline.py

```diff
@@ -1,17 +1,91 @@
-from sentry_streams.examples.word_counter import pipeline, source
-from sentry_streams.pipeline.pipeline import Step
+import pytest
 
+from sentry_streams.pipeline.pipeline import (
+    Filter,
+    KafkaSink,
+    KafkaSource,
+    Map,
+    Pipeline,
+    Step,
+)
 
-def test_register_step() -> None:
+
+@pytest.fixture
+def pipeline() -> Pipeline:
+    pipeline = Pipeline()
+    source = KafkaSource(
+        name="myinput",
+        ctx=pipeline,
+        logical_topic="logical-events",
+    )
+
+    source_2 = KafkaSource(
+        name="myinput2",
+        ctx=pipeline,
+        logical_topic="anotehr-logical-events",
+    )
+
+    filter = Filter(
+        name="myfilter",
+        ctx=pipeline,
+        inputs=[source, source_2],
+        function=simple_filter,
+    )
+
+    _ = Filter(
+        name="myfilter2",
+        ctx=pipeline,
+        inputs=[filter],
+        function=simple_filter,
+    )
+
+    map = Map(
+        name="mymap",
+        ctx=pipeline,
+        inputs=[filter],
+        function=simple_map,
+    )
+
+    map_2 = Map(
+        name="mymap2",
+        ctx=pipeline,
+        inputs=[filter, map],
+        function=simple_map,
+    )
+
+    KafkaSink(
+        name="kafkasink",
+        ctx=pipeline,
+        inputs=[map, map_2],
+        logical_topic="transformed-events",
+    )
+    return pipeline
+
+
+def simple_filter(value: str) -> bool:
```

Comment:
What's the rationale for having these empty tests?

Reply:
This is the function used in the pipeline as application logic.
I advised not to add an implementation because the goal of this test is to validate that the translator works. So the application logic is not important.